types/leaflet.browser.print/index.d.ts

```diff
@@ -0,0 +1,95 @@
+// Type definitions for leaflet.browser.print 1.0
+// Project: https://github.com/Igor-Vladyka/leaflet.browser.print
+// Definitions by: Benjamin Ihrig <https://github.com/ihrigb>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.6
```

Comment:
I don’t see anything here that requires TypeScript 3.6—you should remove this line.